### PR TITLE
[NEW] RMS

### LINKED ISSUE #438: [NEW] Resource Management Service
https://confluence.tsi-dev.otc-service.com/display/DR/RM-4040-RMS-API?sortBy=date&sortOrder=descending
https://docs.otc.t-systems.com/en-us/api/rms/rms_02_0006.html

### PR DESCRIPTION
**Pull request by Aloento**
fix #438

### CODE REVIEW

File: openstack/rms/v1/resources/ListResources.go [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+	Provider string
+	// Specifies the resource type.
+	// Maximum length: 20
+	Type string
+	// Specifies the region ID.
+	// Maximum length: 36
+	RegionId string `q:"region_id,omitempty"`
+	// Specifies the enterprise project ID.
+	// Maximum length: 36
+	EpId string `q:"ep_id,omitempty"`
```
Comment (anton-sidelnikov) [inline openstack/rms/v1/resources/ListResources.go]: call it EnterpriseProjectId

File: openstack/rms/v1/resources/ListResources.go [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+	// Specifies the resource type.
+	Type string `json:"type,omitempty"`
+	// Specifies the region ID.
+	RegionId string `json:"region_id,omitempty"`
+	// Specifies the project ID in OpenStack.
+	ProjectId string `json:"project_id,omitempty"`
+	// Specifies the project name in OpenStack.
+	ProjectName string `json:"project_name,omitempty"`
+	// Specifies the enterprise project ID.
+	EpId string `json:"ep_id,omitempty"`
```
Comment (anton-sidelnikov) [inline openstack/rms/v1/resources/ListResources.go]: same as above

**Review by anton-sidelnikov — CHANGES_REQUESTED** [review]
And tests please

**Review by gtema — CHANGES_REQUESTED** [review]
please no implementation for enterpriseProjectID - this is not present/enabled in OTC - and will not be

**Comment by anton-sidelnikov** [comment]
Will close it, because new service published instead of old rms

**Pull request closed without merging**